### PR TITLE
Added batch script and windows instructions.

### PR DESCRIPTION
**Pull request by jfjlaros**
We might want to replace the script batchjob.py with this script. Once the location is decided, we need to edit the instructions in install.md.

### CODE REVIEW

**Comment by martijnvermaat** [comment]
Yes, this nicely replaces `batchjob.py`.

I think I'll add a short section to the [Mutalyzer developer documentation](http://mutalyzer.readthedocs.org/) about these small utility scripts.

The `install.md` is as you noted not really Mutalyzer specific and more of a one-off, so I would suggest saving that [as a Gist](https://gist.github.com/). Then I can just quickly link to it as an aside from the developer docs.

File: extras/soap-tools/windows/run_batch_job.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    :arg stream output_handle: Opren writable handle for the results.
+    :arg str job_type: Type of the job.
+    :arg str build: Optional build for the Position Converter.
+    """
+    client = Client(URL, cache=None)
+
+    batchfile = input_handle.read().encode("base64")
+    job_id = client.service.submitBatchJob(batchfile, job_type, build)
+    while client.service.monitorBatchJob(job_id):
+        time.sleep(1)
```
Comment (martijnvermaat) [inline extras/soap-tools/windows/run_batch_job.py]: I'd probably up this value a bit and make it a constant definition.

**Comment by martijnvermaat** [comment]
I reworked this a bit based on my earlier comment. Could you save these instructions as a Gist, so I can link to it from our docs?

```
# Running the automated Batch Checker

# Installation
## Python core libraries
Go to the Python [website](https://www.python.org). In the *Downloads* tab,
click the
[Python 2.7.9](https://www.python.org/ftp/python/2.7.9/python-2.7.9.msi)
button. Go to your downloads folder and start the installation by double
clicking on the package *python-2.7.9.msi*.

## Additional python packages
Open a *command prompt* and type the following:

    c:
    cd Python27\Scripts
    pip.exe install suds

## The automated Batch Checker script
Download not yet available.

# Running
Open a *command prompt* and type the following:

    C:\Python27\python.exe run_batch_job.py -h

This will show how to use the script. If, for example, you have a batch file
named *in.txt* on which you want to run the *Name Checker* and you want to
write the output to the file *out.txt*, type the following:

    C:\Python27\python.exe run_batch_job.py in.txt out.txt NameChecker
```

**Comment by jfjlaros** [comment]
Here is the gist: https://gist.github.com/jfjlaros/482fe9f0397e554ed29f

**Comment by martijnvermaat** [comment]
I already made one myself this morning, but I updated it now to point to yours, thanks.

**Pull request merged**